docs/cookbook/infrastructure/flox-store.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Catalog Store
+description: Create a Catalog Store for publishing your own Flox packages
+---
+
+# Setting up a Catalog Store
+
+Publishing your own software to your organization's Flox Catalog requires some
+initial setup, but the process is relatively straightforward. Flox supports
+publishing packages to a Catalog Store, which will exist in an AWS S3 bucket
```

Comment:
Not just AWS S3, but anything S3 compatible like Minio, Backblaze B2, etc

Reply:
OK, I was entirely unaware of that. If we mention these, do we need to expand our discussion of configuring the store?